docs/interpreter/spark.md

```diff
@@ -255,3 +255,29 @@ select * from ${table=defaultTableName} where text like '%${search}%'
 ```
 
 To learn more about dynamic form, checkout [Dynamic Form](../manual/dynamicform.html).
+
+<br />
+### Setting up Zeppelin with Kerberos
+<hr />
+Logical setup with Zeppelin, Kerberos Distribution Center (KDC), and Spark on YARN:
+
+<img src="../assets/themes/zeppelin/img/docs-img/kdc_zeppelin.png">
+
+<b>Configuration Setup</b>
+
+1. On the server that Zeppelin is installed, install Kerberos client modules and configuration, krb5.conf.
+This is to make the server communicate with KDC.
+
+2. Set SPARK_HOME in [ZEPPELIN_HOME]/conf/zeppelin-env.sh to use spark-submit
+(Additionally, you might have to set “export HADOOP_CONF_DIR=/etc/hadoop/conf”)
+
```

Comment:
@sejunra And, I also found that `SPARK_HOME`, `ZEPPELIN_HOME` and `HADOOP_CONF_DIR` are not shown up properly in Zeppelin docs website. 

![screen shot 2016-02-08 at 6 30 27 pm](https://cloud.githubusercontent.com/assets/10060731/12882061/38a143f4-ce92-11e5-8cc7-774ab8628f60.png)

Then, you may use like `SPARK\_HOME` , 
`ZEPPELIN\_HOME` and `HADOOP\_CONF\_DIR` instead of  just using `SPARK_HOME`, `ZEPPELIN_HOME` and `HADOOP_CONF_DIR`.